.github/workflows/tools.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: User space tools/
+
+# 'workflow_dispatch' allows running this workflow manually from the
+# 'Actions' tab
+
+# yamllint disable-line rule:truthy
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  ALL:
```

Comment:
why caps? why not something more informative such as  "build check"

Reply:
A bit cryptic allusion to the default CMake target and its reserved name `ALL`, see #4141 for more background.

How about `Build ALL`?